File: INICIANTE/1010/main.dart
import 'dart:io';

void main(List<String> args) {
  double total = 0;
  for (int i = 0; i < 2; i++) {
    var [code, quantity, value] = stdin.readLineSync()!.split(' ').map(double.parse).toList();
    total += quantity * value;
  }
  print('VALOR A PAGAR: R\$ ${total.toStringAsFixed(2)}');
}

File: INICIANTE/1010/main.js
let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');

let total = 0;
for (let i = 0; i < 2; i++) {
    [code, quantity, value] = lines[i].split(' ').map(Number);
    total += quantity * value;
}
console.log(`VALOR A PAGAR: R$ ${total.toFixed(2)}`);
